config/config.exs

```diff
@@ -2,6 +2,9 @@ import Config
 
 config :elixir, ansi_enabled: true
 
+config :dotcom, :service_rollover_time, ~T[03:00:00]
+config :dotcom, :timezone, "America/New_York"
+
```

Comment:
These aren't alphabetized/can be in the bottom. Thanks for adding these!

Reply:
That file wasn't ever in alphabetical order :) But, I can order it.